corehq/apps/enterprise/tests/test_enterprise_tasks.py

```diff
@@ -43,7 +43,7 @@ def test_email_report_domains_successful(self, mock_send):
         with patch('corehq.apps.enterprise.tasks.get_redis_client', return_val=mock_redis_client):
             email_enterprise_report(self.domain, EnterpriseReport.DOMAINS, self.couch_user)
 
-        expected_title = "Enterprise Dashboard: Project Spaces"
+        expected_title = "Enterprise Console: Project Spaces"
```

Comment:
please revert all the changes in this file. should remain enterprise dashboard as we are just changing the section name